Makefile

```diff
@@ -3,7 +3,6 @@ NODES ?= 10
 PACKAGES ?= api actor command types util version integration/helpers
 LC_ALL = "en_US.UTF-8"
 
-CF_BUILD_VERSION ?= $$(cat BUILD_VERSION) # TODO: version specific
```

Comment:
question(non-blocking): do we still see some value in keeping [build_data.yml](https://github.com/cloudfoundry/cli/blob/main/build_data.yml) up to date? As it was introduced as an eventual alternative to BUILD_VERSION.

Reply:
Good point, even that file can be removed. It was mostly used for min capi tests. Since we don't have any, will remove that as well

Reply:
Done